tests/manual_e2e/basic_test.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Helper function - prints an error message and exits
+exit_error() {
+  echo "Error: $*"
+  exit 1
+}
+
+# Helper function - clean up the WC
+clean_wc() {
+  kubectl delete -f grizzly-e2e-wc.yaml
+  rm grizzly-e2e-wc.yaml
+}
+
+# Helper function - checks the existence of the cm and secret for either alloy or prometheus-agent
+check_configs() {
+  echo "Checking if the corresponding $1-$2 has been created"
+  local config
+
+  [[ "$2" == "config" ]] \
+    && config=$(kubectl get configmap -n org-giantswarm ollyoptest-$1-$2) || config=$(kubectl get secret -n org-giantswarm ollyoptest-$1-$2)
+
+  [[ -z "$config" ]] && echo "$1-$2 not found" || echo "$1-$2 found. Test succeeded"
+}
+
+main() {
+  [[ -z "$1" ]] && exit_error "Please provide the installation name as an argument"
+
+  # Logging into the specified installation to perform the tests
+  tsh kube login $1
+
+  echo "Checking if observability-operator app is in deployed state"
+
+  deployed=$(kubectl get app -n giantswarm observability-operator -o yaml | yq .status.release.status)
+
+  [[ "$deployed" != "deployed" ]] \
+    && exit_error "observability-operator app is not in deployed state. Please fix the app before retrying" || echo "observability-operator app is indeed in deployed state"
+
+  echo "Creating WC"
+
+  # Getting latest Giant Swarm release version
+  toUseRelease="$(kubectl gs get releases -o template='{{range .items}}{{.status.ready}}/{{.metadata.name}}
+  {{end}}' | sed -ne 's/false\/aws-//p' | sort -V | tail -1)"
+
+  # Creating WC template and applying it
+  kubectl gs template cluster --provider capa --name ollyoptest --organization giantswarm --description "observability-operator e2e tests" --release $toUseRelease > grizzly-e2e-wc.yaml
+  kubectl create -f grizzly-e2e-wc.yaml
+
+  echo "WC named 'ollyoptest' created. Waiting for it to be ready"
+
+  sleep 1200
```

Comment:
From what I know we should be able to tell if the cluster is ready by looking at both the cluster cr state and the state of the apps for this cluster. This + a timeout would be more robust than just a sleep.

Reply:
I first used the `kubectl wait --for=condition=Ready=true` command for the cluster but it wasn't working as intended. But I can sure try to find something with the same goal.

Reply:
@TheoBrigitte I removed the huge sleep and used `kubectl wait` command instead.